### ISSUE TITLE
@private should be empty.

### ISSUE THREAD

**Issue by 5saviahv** [gajus/eslint-plugin-jsdoc#670]
## Expected behavior
Not sure is it bug but "empty-tags" error is shown.

## Actual behavior
I am not sure, but I believe it wasnt there before, I made `npm update` yesterday.

Strangely error disappears if I delete empty line after `@private` statement.

## ESLint Config

```js
{
  "env": {
    "node": true
  },
  "plugins": ["jsdoc"],
  "extends": [
    "eslint:recommended",
    "plugin:jsdoc/recommended",
    "plugin:jest/recommended",
    "plugin:node/recommended",
    "prettier"
  ],
  "globals": { "Set": true, "Symbol": true },
  "rules": {
    "array-callback-return": [
      2,
      {
        "allowImplicit": true
      }
    ],
    "block-scoped-var": 2,
    "eqeqeq": [
      2,
      "always",
      {
        "null": "ignore"
      }
    ],
    "no-lonely-if": 2,
    "no-proto": 2,
    "curly": [2, "multi-line"],
    "one-var": [2, "never"],
    "no-else-return": [
      2,
      {
        "allowElseIf": false
      }
    ],
    "no-shadow": 2,
    "no-unused-expressions": 2,
    "no-useless-call": 2,
    "no-use-before-define": [2, "nofunc"],
    "no-void": 2,
    "yoda": 2,
    "strict": 2,

    "jsdoc/require-jsdoc": 0,
    "jsdoc/check-param-names": 2,
    "jsdoc/check-tag-names": 2,
    "jsdoc/check-types": 2,
    "jsdoc/newline-after-description": 2,
    "jsdoc/require-description-complete-sentence": 2,
    "jsdoc/require-hyphen-before-param-description": 2,
    "jsdoc/require-param-description": 2,
    "jsdoc/require-param-name": 2,
    "jsdoc/require-param-type": 2,
    "jsdoc/require-param": 2,
    "jsdoc/valid-types": 2,

    "node/no-unsupported-features/es-builtins": 0, // TODO
    "node/shebang": 0
  }
}
```

## ESLint sample

```js
/**
 * Create an array.
 *
 * @private
 * 
 * @param {string[]} [elem] - Elements to make an array of.
 * @param {boolean} [clone] - Optionally clone nodes.
 * @returns {string[]} The array of nodes.
 */


// Format JS code here
```

## Environment

- Node version: v12.19.0
- ESLint version 7.17.0
- `eslint-plugin-jsdoc` version: 31.0.3

##

**Comment by achsoftathome** [comment]
It can appear it you use /** **/

Warning:
/** Application boot function.
* @async
* @private **/

No warning:
/** Application boot function.
* @async
* @private */

**Comment by brettz9** [comment]
I guess we do have `no-multi-asterisks` to report that already, so perhaps we could allow an exception here for final asterisks...
